ui/src/config.js

```diff
@@ -45,6 +47,21 @@ export function createToken(party) {
         return undefined;
     }
 }
+export function createTokenAll(party) {
```

Comment:
```suggestion
export function createTokenFor(party) {
```
perhaps I'm wrong, I'm just having trouble understanding the `All` part

Reply:
I've switched over to getting the dabl access_tokens from the participants.json file, there is no longer an uploader.